manifests/bosh-manifest/deployments/040-collectd.yml

```diff
@@ -0,0 +1,56 @@
+---
+releases:
+- name: collectd
+  sha1: 5e5c196f9c95fe12eb17bba0f40fdaecaddd6424
+  url: https://github.com/alphagov/paas-collectd-boshrelease/releases/download/0.4/collectd-0.4.tgz
```

Comment:
> I think it should be instead described in the repo why we have a fork of it (e.g. in readme). It seems a bit odd to have to search through commits in other repos hoping for explanation.

If we did that and there was an intention to merge changes upstream we will have to use some sort of branching to keep those README edits out of any pull requests we raise.